app/initializers/register-select-helper.js

```diff
@@ -0,0 +1,13 @@
+import ENV from '../config/environment';
+import registerSelectHelper from 'emberx-select/helpers/register-select-helper';
```

Comment:
I think you can do this with [broccoli-stew's rm](https://github.com/stefanpenner/broccoli-stew/blob/master/lib/rm.js#L40) in an addon hook: http://stackoverflow.com/questions/30686368/exclude-folders-from-builds-in-brocfile/30741440#30741440